docs/embedded/concepts/app-concepts/containertypes.md

```diff
@@ -30,33 +32,19 @@ New-SPOContainerType
 [ -OwningApplicationId <OwningApplicationId>]
 ```
 
-After a Trial Container Type is created, customers can create up to five active Containers in the trial Container Type. Trial Containers are provided with 1 GB of storage space. The Trial Container Type expires after 30 days, resulting in the loss of access to all Containers created using that Container Type. Customers can create a new Trial Container Type, but doing so requires the deletion of ALL existing Trial Containers.
-
-> [!NOTE]
-> The Trial Container Type is the only Container Type that is restricted to the Partner (creating) tenant and cannot be shared with other consuming tenants.
->
-> It is not possible to convert a Trial Container Type and/or the associated Containers to a Standard Container Type.
-
-## Standard Container Types
-
-Standard Container Types are associated with a billing profile in [Azure portal](https://portal.azure.com) when they're created. The billing profile includes an Azure Subscription ID and a Region Group. There are two prerequisites to creating s Standard Container:
+The following restrictions are applied to Container Type in the Trial status:
+1. Up to five active Container instances of the Container Type can be created.
```

Comment:
Markdown numbered bullets should always use 1.... don't use explicit numbers. Let the rendering engine do the numbering for you.

Bullets should be separated by a newline... in other words, there should be a blank line between lines 35 & 36